### ISSUE TITLE
Suggestion: improve feedback about installed Xdebug/Pcov versions

### ISSUE THREAD

**Issue by jrfnl** [shivammathur/setup-php#610]
**Describe the feature**

Currently, when using the `coverage` key, the feedback shown to the user will be along the lines of:
```yaml
      - name: Install PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: ${{ matrix.php }}
          coverage: xdebug
```
```
==> Setup Coverage
✓ xdebug Xdebug enabled as coverage driver
```

I presume this is similar when using PCov.

I believe it would be useful to be able to see at a glance _which_ Xdebug (or Pcov) version was installed and would like to suggest adjusting the feedback to include this information, like so:

```
==> Setup Coverage
✓ xdebug Xdebug x.x.x enabled as coverage driver
```

**Version**
- [x] I have checked releases, and the feature is missing in the latest patch version of `v2`.

**Underlying issue**
If/when something is going wrong with the code coverage recording, the Xdebug/Pcov version used is generally needed in the debugging process.

**Describe alternatives**
For Xdebug, it's easy enough to retrieve the information by adding an extra step, but IMO that's an unnecessary extra step and it would be helpful if the information would always be available in the feedback from `setup-php`.
```yaml
      - name: "DEBUG: Show version details"
        run: php -v
```

**Are you willing to submit a PR?**
If you're open to the idea, I'd be happy to attempt a PR for this, though I haven't looked at the implementation details (yet).

**Comment by shivammathur** [comment]
@jrfnl 

Yes, adding `Xdebug` and `PCOV` versions for coverage input would be great.

For implementing this `phpversion` can be used.
https://www.php.net/manual/en/function.phpversion.php

You can define `extensions.getExtensionVersion` function in `extensions.ts` and call that in `coverage.ts`.

**Comment by jrfnl** [comment]
> You can define `extensions.getExtensionVersion` function in `extensions.ts` and call that in `coverage.ts`.

@shivammathur Thanks for the pointers, but I have to admit that looking at the code in those files, I'm out of my depth in this instance. Sorry. Happy to help in other ways, but I think I need to leave this one for someone who actually knows what they are doing ;-)

**Comment by shivammathur** [comment]
Added in https://github.com/shivammathur/setup-php/commit/050cb8061ba059cc26c52e2195ed2541f6b80d91

**Comment by jrfnl** [comment]
@shivammathur You are an absolute star! Thank you so much.

**Comment by jrfnl** [comment]
Would you like me to close the issue ? Or do you leave it open until the feature has been released ?

**Comment by shivammathur** [comment]
I will close it when I create the new release, most likely on Monday next week.

**Comment by shivammathur** [comment]
Released in [2.20.0](https://github.com/shivammathur/setup-php/releases/tag/2.20.0)
